src/main/java/com/work/management/web/rest/controller/EmployeeController.java

```diff
@@ -61,7 +61,7 @@
   @RequestMapping(method = RequestMethod.PUT, value = "/bulkUpdate", produces = "application/json; charset=UTF-8")
   @ApiOperation(value = "Bulk updates on employees details")
   ResponseEntity<List<Employee>> employeeBulkUpdate(
-      @Valid @RequestBody @ApiParam(value = "Bulk update on employee details", required = true) BulkEmployeeRequest bulkEmployeeRequest) {
+      @Valid  @ApiParam(value = "Bulk update on employee details", required = true) @RequestBody BulkEmployeeRequest bulkEmployeeRequest) {
     return new ResponseEntity<>(
```

Comment:
What's the need for this change?

Reply:
just for the sake of testing purpose but now i have back to original